### PR TITLE
Make API much faster for single-org deployments

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

- Removes some slow org-checking when there's only one org.  
- Gets rid of unused FE tracking  
- Cuts some cruft out of some queries 

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [x] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by michielderoos** [comment]
[Remove unused FE tracking](https://trello.com/c/dYwvCbZ1/959-remove-unused-fe-tracking)

**Comment by michielderoos** [comment]
[Optimize custom attribute options generation for metrics/filters endpoint](https://trello.com/c/f9oTLu0K/955-optimize-custom-attribute-options-generation-for-metrics-filters-endpoint)

**Comment by michielderoos** [comment]
[Optimize organisation membership filter](https://trello.com/c/c39Z4Jnf/956-optimize-organisation-membership-filter)

File: app/server/models/utils.py [hunk 1]
```diff
@@ -48,19 +48,22 @@ def no_expire():
...
     show_deleted = getattr(g, "show_deleted", False) or query._execution_options.get("show_deleted", False)
     show_all = getattr(g, "show_all", False) or query._execution_options.get("show_all", False)
     # We want to support multiple active organizations, but only for select GET requets.
     # This is done through a multi_org flag, very similar to the show_all flag
     multi_org = getattr(g, "multi_org", False) or query._execution_options.get("multi_org", False)
     if show_all and show_deleted:
         return query
-
+    if org_check:
+        return query
```
Comment (tristanhcole) [inline app/server/models/utils.py:64]: `org_check` isn't used anywhere?

File: app/server/templates/index.html [hunk 2]
```diff
@@ -61,49 +61,6 @@
 
       gtag("config", "{{ config.GOOGLE_ANALYTICS_ID }}");
     </script>
-
-    <script type="text/javascript">
```
Comment (tristanhcole) [inline app/server/templates/index.html:65]: image

**Review by tristanhcole — APPROVED** [review]

**Comment by tristanhcole** [comment]
muuuuuuch faster! nice!

File: app/server/models/utils.py (hunk 1 above)
Comment (michielderoos) [reply app/server/models/utils.py:64]: I made up `org_check` specifically for this application

**Comment by michielderoos** [comment]
[Adjust wsgi settings for better pefromance](https://trello.com/c/O0nZ4mc8/957-adjust-wsgi-settings-for-better-pefromance)

**Pull request merged**